src/module.c

```diff
@@ -464,6 +466,99 @@ int GetSingleDocumentCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int
   return REDISMODULE_OK;
 }
 
+int SpellCheckCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
+#define DICT_INITIAL_SIZE 5
```

Comment:
In the future, would `const static size_t` work instead of inline macro definitions?